### PR TITLE
[FEAT] Enhanced Presidio

### PR DESCRIPTION
**Pull request by rmehtany**

### CODE REVIEW

File: api/llm/handler.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ async def create_chat_completion(
 
     orch.privacy.tokenize(session_id=session_id, messages=msgs)
 
+    print(orch.privacy.anonymizer.finder.find_sensitive_data_locations)
+
```
Comment (rmehtany) [inline api/llm/handler.py:79]: Can we remove this print?

File: db/audit_log.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 class AddLogParams:
     session_id: str
     username: str
-    log_type: models.PontusLogtype
+    log_type: Any
```
Comment (rmehtany) [inline db/audit_log.py:25]: Why are we changing the typing here?

File: pkgs/privacy/sensitive_data/__init__.py [hunk 3]
```diff
@@ -11,7 +11,9 @@ def build_sensitive_data_finder(config: "SensitiveDataConfig") -> SensitiveDataF
     match (config.sensitive_data_method):
         case SensitiveDataFinderMethod.presidio:
             return Presidio(threshold=config.threshold, sensitive_data_types=config.sensitive_data_types)
-        case SensitiveDataFinderMethod.presidio_and_addresses:
-            return PresidioAndAddresses(threshold=config.threshold, sensitive_data_types=config.sensitive_data_types)
+        case SensitiveDataFinderMethod.EnahncedPresdio:
+            print("Requires commercial license for commercial usage. ")
+            # TODO: do liscense check
```
Comment (rmehtany) [inline pkgs/privacy/sensitive_data/__init__.py:16]: liscense -> license

File: api/llm/handler.py (hunk 1 above)
Comment (sumants-dev) [reply api/llm/handler.py:79]: done

File: db/audit_log.py (hunk 2 above)
Comment (sumants-dev) [reply db/audit_log.py:25]: my make gen did this not sure why

Comment (sumants-dev) [reply db/audit_log.py:25]: i'll revert

Comment (sumants-dev) [reply db/audit_log.py:25]: actually prob not because this is what sqlc is saying

**Review by sumants-dev — APPROVED** [review]

**Pull request merged**